.github/workflows/scripts/pre-release/references.sh

```diff
@@ -32,3 +33,16 @@ if [[ "$results" != "" ]]; then
     exit 1
 fi
 
+# Verify documentation refers to the most recent release tag
+results=$(
+    find . -name "*.md" -print0 \
+    | xargs -0 grep -Pn "uses: slsa-framework/.*@(?!<|main|$RELEASE_TAG)" \
```

Comment:
could we qualify w/ the repo name slsa-github-generator too? just in case we start using slsa-verifier action

Reply:
Good catch. Done.